uaa/src/test/java/org/cloudfoundry/identity/uaa/integration/ScimUserEndpointsIntegrationTests.java

```diff
@@ -140,6 +143,22 @@ public void createUserSucceeds() throws Exception {
         assertTrue(joe2.isVerified());
     }
 
+    // curl -v -H "Content-Type: application/json" -H "Accept: application/json"
```

Comment:
remove comments?

Reply:
I believe those are helpful in troubleshooting .....it is the test code and given the beast that uaa...any help and directions is good...

Reply:
good point, thanks